lib/src/app/app.dart

```diff
@@ -471,8 +471,8 @@ class AppClient {
   /// Get a list of [Fragment]s in an [Organization]
   ///
   /// For more information, see [Fleet Management API](https://docs.viam.com/appendix/apis/fleet/).
-  Future<List<Fragment>> listFragments(String organizationId, bool showPublic) async {
-    final request = ListFragmentsRequest()
+  Future<List<Fragment>> listFragments(String organizationId, bool showPublic, {List<FragmentVisibility>? fragmentVisibility}) async {
+    final request = ListFragmentsRequest(fragmentVisibility: fragmentVisibility)
```

Comment:
Can you please switch this to the `..fragmentVisiblity = fragmentVisibility` syntax like the other options?

Reply:
changed !

Reply:
ok, actually should be good to go now - had to use addAll since it's an iterable